### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Mayureshd-18**
Fixed some typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [overcast-security.com](https://overcast-security.com/) - We make tracking your external attack surface easy
 - [IPInfo.io](https://ipinfo.io/) - The trusted source for IP address data
 - [IPData.co](https://ipdata.co/) - IP Geolocation and Threat Intelligence API
-- [NetworksDB](https://networksdb.io/) - information about the public IPv4 and IPv6 addresses, networks and domains owned by companies and organisations across the world
+- [NetworksDB](https://networksdb.io/) - information about the public IPv4 and IPv6 addresses, networks and domains owned by companies and organizations across the world
```
Comment (edoardottt) [inline README.md:132]: We can drop this, it's okay to use that term

File: README.md [hunk 2]
```diff
@@ -243,7 +243,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 ### DNS
 
 - [DNSDumpster](https://dnsdumpster.com/) - dns recon & research, find & lookup dns records
-- [Chaos](https://chaos.projectdiscovery.io/#/) - Enhance research and analyse changes around DNS for better insights
+- [Chaos](https://chaos.projectdiscovery.io/#/) - Enhance research and analyze changes around DNS for better insights
```
Comment (edoardottt) [inline README.md:246]: We can drop this, it's okay to use that term

File: README.md [hunk 3]
```diff
@@ -275,15 +275,15 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
...
+- [certificatedetails](https://certificatedetails.com/) - Online certificate viewer. Inspect and download certificates from your browser
 - [FacebookCT](https://developers.facebook.com/tools/ct/search/) - Search for certificates issued for a given domain and subscribe to notifications from Facebook regarding new certificates
 - [certs.io](https://certs.io/) - Search TLS certificates across the internet.
 
 ### WiFi Networks
 
 - [Wigle.net](https://wigle.net/) - Maps and database of 802.11 wireless networks with statistics
 - [wifimap.io](https://www.wifimap.io/) - Connect to all Free WiFi Hotspots using WiFi Map App all over the World!
-- [wificafespots.com](http://www.wificafespots.com/) - Free WiFi Cafe Spots
+- [wificafespots.com](http://www.wificafespots.com/) - Free WiFi Café Spots
```
Comment (edoardottt) [inline README.md:286]: We can drop this, it's okay to use that term

**Review by edoardottt — CHANGES_REQUESTED** [review]
requesting small changes

**Comment by Mayureshd-18** [comment]
@edoardottt made the changes. Please review.
Regards.

File: README.md [hunk 4]
```diff
@@ -130,7 +130,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [IPInfo.io](https://ipinfo.io/) - The trusted source for IP address data
 - [IPData.co](https://ipdata.co/) - IP Geolocation and Threat Intelligence API
 - [NetworksDB](https://networksdb.io/) - information about the public IPv4 and IPv6 addresses, networks and domains owned by companies and organisations across the world
-- [ASNlookup](https://asnlookup.com/) - Quickly lookup updated information about specific Autonomous System Number (ASN), Organization, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data
+- [ASNlookup](https://asnlookup.com/) - Quickly lookup updated information about specific Autonomous System Number (ASN), Organisations, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data
```
Comment (edoardottt) [inline README.md:133]: If you google `asnlookup com`, you'll see its official description. it's not a typo

**Review by edoardottt — CHANGES_REQUESTED** [review]

**Comment by Mayureshd-18** [comment]
You want it with 's' or with the 'z'?

**Comment by edoardottt** [comment]
This is the official description, so we just keep it as it is: `Quickly lookup updated information about specific Autonomous System Number (ASN), Organization, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data`

**Comment by Mayureshd-18** [comment]
Thanks. Made the changes.

File: README.md [hunk 5]
```diff
@@ -130,7 +130,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [IPInfo.io](https://ipinfo.io/) - The trusted source for IP address data
 - [IPData.co](https://ipdata.co/) - IP Geolocation and Threat Intelligence API
 - [NetworksDB](https://networksdb.io/) - information about the public IPv4 and IPv6 addresses, networks and domains owned by companies and organisations across the world
-- [ASNlookup](https://asnlookup.com/) - Quickly lookup updated information about specific Autonomous System Number (ASN), Organization, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data
+- [ASNlookup](https://asnlookup.com/) - Quickly lookup updated information about specific Autonomous System Number (ASN), Organizations, CIDR, or registered IP addresses (IPv4 and IPv6) among other relevant data
```
Comment (edoardottt) [inline README.md:133]: **this is not a typo**

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Mayureshd-18) [reply README.md]: Sorry for misunderstanding your request.

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**